site/docs/master/velero-install.md

```diff
@@ -0,0 +1,46 @@
+# Velero Install CLI
+
+This document serves as a guide to using the `velero install` CLI command to install `velero` server components into your kubernetes cluster.
+
+_NOTE_: `velero install` will, by default, use the version information to determine the version of of the server compoents to deploy. This behavior may be overridden by using the `--image` flag. Refer to [Building Server Component Container Images For Server Component][1].
+
+## Usage
+
+This section explains some of the basic flags supported by the `velero install` CLI command. For a complete explaination of the flags, please run `velero install --help`
+
+```(bash)
+velero install \
+    --plugins <PLUGIN_CONTAINER_IMAGE [PLUGIN_CONTAINER_IMAGE]>
+    --provider <YOUR_PROVIDER> \
+    --bucket <YOUR_BUCKET> \
+    --secret-file <PATH_TO_FILE> \
+    --velero-pod-cpu-request <CPU_REQUEST> \
+    --velero-pod-mem-request <MEMORY_REQUEST> \
+    --velero-pod-cpu-limit <CPU_LIMIT> \
+    --velero-pod-mem-limit <MEMORY_LIMIT> \
+    [--use-restic] \
+    [--restic-pod-cpu-request <CPU_REQUEST>] \
+    [--restic-pod-mem-request <MEMORY_REQUEST>] \
+    [--restic-pod-cpu-limit <CPU_LIMIT>] \
+    [--restic-pod-mem-limit <MEMORY_LIMIT>]
+```
+
+In the about usage explanation:
```

Comment:
I'd remove this sentence and the numbers from the following two.

Reply:
Not sure I follow the comment. Can you please clarify?

Reply:
Are you suggesting that I remove the numbered list?

Reply:
To rephrase:

I'd delete L28 and take the `1.` off of L30 and L31

Reply:
Thanks for clarifying!